src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
Copied from Auth0.

Reply:
I don't see anything Auth0-specific in here so I'm pretty sure you can just kill it and not bother importing it above ... this is just `create-react-app` boilerplate

Reply:
I didn't see these comments until after I submitted my review, sorry!

Reply:
@macfarlandian as far as I can tell, this doesn't exist in the repo anywhere and needs to be invoked where I've invoked it. Let me know if I'm misunderstanding!

Reply:
it doesn't exist anywhere, but I think all this code is doing is importing this module and immediately disabling it (which is also part of the CRA boilerplate code). Unless taking it out (including killing that import and any references to it) causes the auth to stop working, it shouldn't be needed. 

That said it shouldn't be hurting anything either, it's just cruft, so I'd also be fine with just leaving a TODO somewhere to investigate and clean up so we can move on to more critical stuff